src/java.base/share/classes/java/lang/foreign/SymbolLookup.java

```diff
@@ -283,7 +284,8 @@ static SymbolLookup libraryLookup(String name, Arena arena) {
      * @throws IllegalStateException if {@code arena.scope().isAlive() == false}
      * @throws WrongThreadException if {@code arena} is a confined arena, and this method
      *         is called from a thread {@code T}, other than the arena's owner thread
-     * @throws IllegalArgumentException if {@code path} does not point to a valid library
+     * @throws IllegalArgumentException if {@code path} is not in the default file system
+     *         or if {@code path} does not point to a valid library
```

Comment:
Please split this into separate `@throws IllegalArgumentException` tags, as that is what we do everywhere else in the FFM API.

Reply:
It depends on the wording. If changed to something like "not a path to a valid library in the default file system" then it avoid describing it as two cases.